addons/barcodes/models/barcode_events_mixin.py

```diff
@@ -11,7 +11,7 @@ class BarcodeEventsMixin(models.AbstractModel):
 
     _name = 'barcodes.barcode_events_mixin'
 
-    _barcode_scanned = fields.Char("Barcode Scanned", help="Value of the last barcode scanned.", store=False)
+    _barcode_scanned = fields.Char(string="Barcode Scanned", help="Value of the last barcode scanned.", store=False)
```

Comment:
Hey man , you are ??? please check your sense, No need to increase diff , without string it work Waw ammmmzzzzed.
I think you when you code >>> :laughing: 
![codelike](https://cloud.githubusercontent.com/assets/14940179/12079666/69f5e154-b1fe-11e5-93c9-ce03d13866cf.jpg)